src/phantasm-renderer/backend/lib/murmur_hash.hh

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+#include <cstdint>
+
+namespace pr::backend::hash
+{
+// UNUSED
+void murmurhash3_x64_128(const void* key, int len, uint32_t seed, void* out);
+}

```

Comment:
hm this might be good to add to `cc` as well

Reply:
yes. From what ive seen, xx hash is even faster, but its implementation isn't as nice and compact as this one.
https://github.com/Cyan4973/xxHash